### ISSUE TITLE
[Bug]: GroqException - property 'stop_sequences' is unsupported

### ISSUE THREAD

**Issue by Clad3815** [BerriAI/litellm#4483]
### What happened?

If we use the "stop_sequences" with groq we have this error, even with `drop_params: true`. Based on the docs, the name is `stop: A stop sequence is a predefined or user-specified text string that signals an AI to stop generating content, ensuring its responses remain focused and concise.` https://console.groq.com/docs/text-chat

### Relevant log output

```shell
litellm-1  |   File "/usr/local/lib/python3.11/site-packages/litellm/utils.py", line 7631, in exception_type
litellm-1  |     raise e
litellm-1  |   File "/usr/local/lib/python3.11/site-packages/litellm/utils.py", line 5932, in exception_type
litellm-1  |     raise BadRequestError(
litellm-1  | litellm.exceptions.BadRequestError: litellm.BadRequestError: BadRequestError: GroqException - property 'stop_sequences' is unsupported, did you mean 'top_logprobs'? LiteLLM Retried: 2 times, LiteLLM Max Retries: 3
litellm-1  |
```

**Comment by ishaan-jaff** [comment]
@Clad3815 the param is `stop` not `stop_sequences` 

<img width="385" alt="Screenshot 2024-07-01 at 8 55 30 PM" src="https://github.com/BerriAI/litellm/assets/29436595/ecaa6570-2bcd-4741-bc6c-28ff0e970031">

**Comment by ishaan-jaff** [comment]
@Clad3815 feel free to re-open if it's raising an error when you pass `stop` to litellm.completion

**Comment by Clad3815** [comment]
I said it in my first message. I don't understand your answer, the goal is not to make LiteLLM work with every LLM just by changing the model in our call ?

I'm using the OpenAI sdk with LiteLLM as proxy, and so I can't use "stop". We have to use the OpenAI format, which is "stop_sequences" and which work with others providers also.

**Comment by Clad3815** [comment]
@ishaan-jaff Did I misunderstand something ?
